packages/suite/src/hooks/suite/useTranslation.ts

```diff
@@ -2,16 +2,22 @@ import { useCallback } from 'react';
 import { useIntl } from 'react-intl';
 import messages from '@suite/support/messages';
 import { ExtendedMessageDescriptor } from '@suite-types';
+import { PrimitiveType } from '@suite/types/utils';
 
-type PrimitiveType = string | number | boolean | Date | null | undefined;
+export type TransFunction = (
```

Comment:
please do not use shortcuts Trans => Translation

Reply:
done